contracts/ATM/ATMGovernance.sol

```diff
@@ -0,0 +1,163 @@
+pragma solidity 0.5.17;
+
+// External Libraries
+import "@openzeppelin/contracts/lifecycle/Pausable.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+// Common
+import "../util/AddressArrayLib.sol";
+import "../util/AddressLib.sol";
+
+// Contracts
+import "@openzeppelin/contracts/access/roles/SignerRole.sol";
+
+// Interfaces
+import "./IATMGovernance.sol";
+
+/**
+    @notice This contract is used to modify Risk Settings, CRA or DataProviders for a specific ATM.
+    @author develop@teller.finance
+ */
+contract ATMGovernance is Pausable, SignerRole, IATMGovernance {    
+    // TODO: REMOVE PAUSABLE
+
+    using AddressArrayLib for address[];
+    using AddressLib for address;
+    using Address for address;
+
+    // List of general ATM settings. We don't accept settings equal to zero.
+    // Example: supplyToDebtRatio  => 5044 = percentage 50.44
+    // Example: supplyToDebtRatio => 1 = percentage 00.01
+    mapping(bytes32 => uint256) public generalSettings;
+
+
+    // List of Market specific Asset settings on this ATM
+    // Asset address => Asset setting name => Asset setting value
+    mapping(address => mapping(bytes32 => uint256)) public assetMarketSettings;
```

Comment:
What would be some examples of this mapping?

Reply:
As I saw in the unit tests, here is where we can configure the supplyToDebtRatio per market.
My question is how we define the market. It is a pair (borrowed/lent token and collateral token. Ex: DAI/ETH or USDC/ETH). But we only have one asset in this mapping.

Reply:
As of today supplyToDebtRadio is one per ATM so it's configured as a General Setting (generalSettings['supplyToDebtRadio']).

Then, the assets we configure per market are assetMarketSettings[asset address][setting name] = value and I'll add some examples in the comments also.